totalRP3_Extended/aura/aura.xml

```diff
@@ -0,0 +1,244 @@
+			<Layer level="BACKGROUND">
+				<Texture parentKey="BottomLeft" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="BOTTOMLEFT"/>
+					</Anchors>
+					<TexCoords top="0.03125" left="0.015625" bottom="0.53125" right="0.265625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="BottomRight" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="BOTTOMRIGHT"/>
+					</Anchors>
+					<TexCoords top="0.03125" left="0.265625" bottom="0.53125" right="0.015625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopLeft" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="TOPLEFT"/>
+					</Anchors>
+					<TexCoords top="0.53125" left="0.015625" bottom="0.03125" right="0.265625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopRight" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="TOPRIGHT"/>
+					</Anchors>
+					<TexCoords top="0.53125" left="0.265625" bottom="0.03125" right="0.015625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopEdge" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.TopLeft" relativePoint="TOPRIGHT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.TopRight" relativePoint="BOTTOMLEFT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="BottomEdge" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.BottomLeft" relativePoint="TOPRIGHT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.BottomRight" relativePoint="BOTTOMLEFT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="Center" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.TopLeft" relativePoint="BOTTOMLEFT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.BottomRight" relativePoint="TOPRIGHT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+			</Layer>
+		</Layers>
+		<Scripts>
+			<OnEnter>
+				self.Text:Show();
+				self.BottomLeft:Show();
+				self.BottomRight:Show();
+				self.TopLeft:Show();
+				self.TopRight:Show();
+				self.TopEdge:Show();
+				self.BottomEdge:Show();
+				self.Center:Show();
+			</OnEnter>
+			<OnLeave>
+				self.Text:Hide();
+				self.BottomLeft:Hide();
+				self.BottomRight:Hide();
+				self.TopLeft:Hide();
+				self.TopRight:Hide();
+				self.TopEdge:Hide();
+				self.BottomEdge:Hide();
+				self.Center:Hide();
+			</OnLeave>
+			<OnLoad>
+				self.UpdatePosition = function(self)
```

Comment:
I'm gonna make a general blanket comment here - move all of these inline script bodies to a mixin defined in a Lua file, eg.

```lua
TRP3_ExtendedAuraFrameMixin = {};

function TRP3_ExtendedAuraFrameMixin:OnLoad()
    ...
end

function TRP3_ExtendedAuraFrameMixin:UpdatePosition()
    ...
end
```

And then associate the mixin with the template by adding the `mixin="TRP3_ExtendedAuraFrameMixin"` attribute and using method refs for these script handlers:

```xml
<OnLoad method="OnLoad"/>
<OnDragStart method="OnDragStart"/>
...
```